tests/main/views/test_buyers.py

```diff
@@ -241,23 +239,27 @@ def test_flash_message_shown_if_brief_has_just_been_updated(self, data_api_clien
 class TestBuyerRoleRequired(BaseApplicationTest):
     def test_login_required_for_buyer_pages(self):
         with self.app.app_context():
-            res = self.client.get('/buyers')
+            res = self.client.get(self.briefs_dashboard_url)
             assert res.status_code == 302
-            assert res.location == 'http://localhost/user/login?next=%2Fbuyers'
+            assert res.location == 'http://localhost/user/login?next={}'.format(
+                self.briefs_dashboard_url.replace('/', '%2F')
+            )
 
     def test_supplier_cannot_access_buyer_pages(self):
         with self.app.app_context():
             self.login_as_supplier()
-            res = self.client.get('/buyers')
+            res = self.client.get(self.briefs_dashboard_url)
```

Comment:
This test should probably cover both pages: the main (new) buyers dashboard, and the dos buyers dashboard.